.github/workflows/test.yaml

```diff
@@ -83,6 +83,10 @@ jobs:
             - name: Allow unstable dependencies
               run: composer config minimum-stability dev
 
+            - name: Remove dev dependencies not compatible with Symfony 6.0
+              if: matrix.symfony-require == '6.0.*'
+              run: composer remove vimeo/psalm psalm/plugin-phpunit psalm/plugin-symfony weirdan/doctrine-psalm-plugin --dev --no-interaction
```

Comment:
Psalm is not compatible with sf 6, but we don't need it to run test, so we can remove it for now on the test build. When support is added, we can remove those lines and it should work the same.